src/Symfony/Component/Validator/Constraints/CountValidator.php

```diff
@@ -21,6 +22,13 @@
  */
 class CountValidator extends ConstraintValidator
 {
+    private $expressionLanguage;
+
+    public function __construct(ExpressionLanguage $expressionLanguage = null)
+    {
+        $this->expressionLanguage = $expressionLanguage;
+    }
```

Comment:
FrameworkBundle should register the validator as a service now and inject a configured expression language instance.

Reply:
> @zolex Are you still interested in finishing this PR?

will continue today.

Reply:
I have checked the other components that are using the expression language and it is not using dependency injection anywhere. the only thing that is done in some places is allowing to add `ExpressionFunctionProviderInterface`s to the class that creates the expression language instance and passes the additional functions to the constructor, but it's not implemented like this everywhere, sometimes they just create the instance and that's it.